base/Makefile

```diff
@@ -12,13 +12,13 @@ else
 CPP_STDOUT = $(CPP) -E
 endif
 
-all: pcre_h.jl errno_h.jl build_h.jl.phony fenv_constants.jl file_constants.jl uv_constants.jl version_git.jl.phony
+all: errno_h.jl build_h.jl.phony fenv_constants.jl file_constants.jl uv_constants.jl version_git.jl.phony
```

Comment:
Just temporary, since the pcre_h.jl autogenerator was failing on pcre2.h - it would run without error, but output a blank pcre_h.jl. I'm looking into it.